### PR TITLE
Topics: separate OP_CAT into its own topic page

### PR DESCRIPTION
**Pull request by harding**
It was previously a subsection of OP_CSFS.  This separation was suggested in #721

### CODE REVIEW

**Comment by bitschmidty** [comment]
Pushed a commit with some other op_cat references updated. It seemed to me those should go to the new op_cat topic but if the csfs topic is better, feel free to drop accordingly

**Comment by harding** [comment]
LGTM, thanks @bitschmidty !

File: _topics/en/op_cat.md [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+  **OP_CAT** was originally an opcode in Bitcoin.  It was disabled in
+  2010 but slight variations on it are frequently proposed to be
+  added to Bitcoin using a soft fork.
+
+---
+Both the original `OP_CAT` and the new proposals for it
+concatenate two elements on the stack into a single element.  For
+example, the following script:
+
+    <0xB10C> <0xCAFE> OP_CAT
```
Comment (bitschmidty) [inline _topics/en/op_cat.md:82]: 0xb10c hehe

**Review by bitschmidty — APPROVED** [review]
ACK

**Pull request merged**